### PR TITLE
i18n(fr): Update `guides/routing.mdx` from #9769

### PR DESCRIPTION
**Pull request by thomasbnt**
#### Description (required)
Update `guides/routing.mdx` from #9769

#### Related issues & labels (optional)

- Suggested label: i18n

I'm participating to the **Hacktoberfest 2024**!

### CODE REVIEW

File: src/content/docs/fr/guides/routing.mdx [hunk 1]
```diff
@@ -392,6 +393,12 @@ Dans l'exemple ci-dessus, voici quelques exemples de la manière dont les règle
 - `pages/post/[...slug].astro` - Construira `/post/1/2`, `/post/a/b/c`, etc. Mais pas `/post/create`, `/post/1`, `/post/abc` etc.
 - `pages/[...slug].astro` - Construira `/abc`, `/xyz`, `/abc/xyz`, etc. Mais pas `/posts/create`, `/posts/1`, `/posts/abc`, etc.
 
+### Routes réservées
+Les routes internes ont la priorité sur les routes définies par l'utilisateur ou par l'intégration, car elles sont nécessaires au déroulement des fonctionnalités d'Astro. Les routes réservées d'Astro sont les suivantes :
```
Comment (ArmandPhilippot) [inline src/content/docs/fr/guides/routing.mdx:397]: nit: adds missing blank lines
```suggestion
### Routes réservées

Les routes internes ont la priorité sur les routes définies par l'utilisateur ou par l'intégration, car elles sont nécessaires au déroulement des fonctionnalités d'Astro. Les routes réservées d'Astro sont les suivantes :

```

File: src/content/docs/fr/guides/routing.mdx [hunk 2]
```diff
@@ -392,6 +393,12 @@ Dans l'exemple ci-dessus, voici quelques exemples de la manière dont les règle
...
+- `_astro/`: Fournit toutes les ressources statiques au client, y compris les documents CSS, les scripts client intégrés, les images optimisées et toutes les ressources Vite.
```
Comment (ArmandPhilippot) [inline src/content/docs/fr/guides/routing.mdx:398]: nit: missing space after colon
```suggestion
- `_astro/` : Fournit toutes les ressources statiques au client, y compris les documents CSS, les scripts client intégrés, les images optimisées et toutes les ressources Vite.
```

File: src/content/docs/fr/guides/routing.mdx [hunk 3]
```diff
@@ -392,6 +393,12 @@ Dans l'exemple ci-dessus, voici quelques exemples de la manière dont les règle
...
+- `_server_islands/`: Servir les composants dynamiques différés dans une [Île de serveur](/fr/reference/configuration-reference/#experimentalserverislands);
```
Comment (ArmandPhilippot) [inline src/content/docs/fr/guides/routing.mdx:399]: nit: missing space after colon + replaces the infinitive form to be consistent
```suggestion
- `_server_islands/` : Sert les composants dynamiques différés dans une [Île de serveur](/fr/reference/configuration-reference/#experimentalserverislands);
```

File: src/content/docs/fr/guides/routing.mdx [hunk 4]
```diff
@@ -392,6 +393,12 @@ Dans l'exemple ci-dessus, voici quelques exemples de la manière dont les règle
...
+- `_actions/`: Serves any defined [actions](/fr/guides/actions/).
```
Comment (ArmandPhilippot) [inline src/content/docs/fr/guides/routing.mdx:400]: missing translation:
```suggestion
- `_actions/` : Sert toutes les [actions](/fr/guides/actions/) définies.
```

**Review by ArmandPhilippot — CHANGES_REQUESTED** [review]
Hi @thomasbnt! You missed one translation... the rest is mostly nitpicking!

**Comment by thomasbnt** [comment]
> Hi @thomasbnt! You missed one translation... the rest is mostly nitpicking!

Thanks for theses reviews!

**Review by ArmandPhilippot — APPROVED** [review]
LGTM!

**Review by yanthomasdev — APPROVED** [review]
LGTM! Good work!

**Pull request merged**